### PR TITLE
Add python arg

### PR DESCRIPTION
**Pull request by joakim-hove**
Downstream
https://github.com/OPM/opm-material/pull/414
https://github.com/OPM/opm-simulators/pull/2501

See https://github.com/OPM/opm-simulators/pull/2501 for more details

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this opm-material=414 opm-simulators=2501 please

**Comment by joakim-hove** [comment]
jenkins build this opm-material=414 opm-simulators=2501 please

**Comment by joakim-hove** [comment]
jenkins build this opm-material=414 opm-simulators=2501 please

**Comment by joakim-hove** [comment]
jenkins build this opm-material=414 opm-simulators=2501 please

**Comment by joakim-hove** [comment]
jenkins build this opm-material=414 opm-simulators=2501 please

File: opm/parser/eclipse/Python/Python.hpp [hunk 1]
```diff
@@ -70,11 +70,11 @@ class EclipseState;
   True     |  OFF               |  *                  | { }
```
Comment (atgeirr) [inline opm/parser/eclipse/Python/Python.hpp]: I do not understand what the third column means: is it "Have we already constructed an instance?"

What does the * mean?

For TRY, I do not understand the Result column. Assuming the third column meaning as above, when we already have constructed an instance, we suddenly get no instance? I would think that you would refer to the existing one? That is, assuming `{}` means "no Python for you", and `{ Python }` means "yes, Python can be used", and the `if (python)` check in the example above returns `{false|true}` respectively.

**Comment by joakim-hove** [comment]
jenkins build this opm-material=414 opm-simulators=2501 please

**Pull request merged**